tools/ci_build/github/azure-pipelines/templates/win-ci-vs-2019.yml

```diff
@@ -62,7 +62,7 @@ jobs:
     OnnxRuntimeBuildDirectory: '$(Build.BinariesDirectory)'
     DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
     setVcvars: true
-    ALLOW_RELEASED_ONNX_OPSET_ONLY: '0'
+    ALLOW_RELEASED_ONNX_OPSET_ONLY: '1'
```

Comment:
Why?

Reply:
- currently onnx-script need a feature is this PR to unblock its development. This PR does not implement cpu computation for new/and update kernels.
- we like to integrate with ONNX more frequently and gradually instead of doing it once after ONNX is released. Which means it may not have all cpu kernels implemented.
- it does make sense to only validate released ops.

Reply:
Please notify Pranav and Ashwini because this is a significant change. Also, it means that ORT will not run any ONNX test before it is released. So, ORT cannot be used to validate an ONNX release candidate.